ofrak_core/ofrak/gui/server.py

```diff
@@ -475,6 +563,10 @@ async def search_for_vaddr(self, request: Request) -> Response:
     async def add_tag(self, request: Request) -> Response:
         resource = await self._get_resource_for_request(request)
         tag = self._serializer.from_pjson(await request.json(), ResourceTag)
+        script_str = """
+        {resource}.add_tag(tag)
```

Comment:
Should this be something like the following? Right now it adds a tag from a variable that doesn't exist.

```suggestion
        {resource}.add_tag({tag.__name__})
```